requirements.yml

```diff
@@ -12,7 +12,8 @@ collections:
     version: ee_req_install
     type: git
   - name: community.vmware  # has requirements.txt, but may add pyvcloud
-  - name: ovirt.ovirt
+  - name: https://github.com/shanemcd/ovirt-ansible-collection
+    type: git
```

Comment:
So, for this. I will update our build jobs, to allow us to use depends-on for this too.

The trick is, we need to either allow for git contents inside galaxy layer, like we did with ZUUL_SIBLINGS, or pre-compiled tarballs.

Either way, soon we won't have to do this for CI. Which I am excited for.